docs/source/working_with_aiida/index.rst

```diff
@@ -2,10 +2,16 @@
 Command line interface
 ======================
 
-The main way of interacting with AiiDA is through a command line interface tool called ``verdi``.
-Below you will find an overview of all the commands that are available with a link to a more detailed explanation of their usage and available parameters.
-But before you dive in, take a few minutes to read the :ref:`general concepts<cli_concepts>` that apply to the entire interface of ``verdi``.
-This will make understanding and using ``verdi`` a lot easier!
+One way of interacting with AiiDA is through the ``verdi`` command line interface.
```

Comment:
Either here or in the other file, could you add the common acronym (CLI) after writing out command line interface? It is a kind favour to the reader, I think, but obviously not mandatory :)

Reply:
done!